README.md

```diff
@@ -0,0 +1,31 @@
+
+**Figure 3.** Summary tree of the genus *Ganoderma* inferred from ML analysis, based on ITS sequence data (main dataset, DS; Table 2). Thick lines represent ML bootstrap values (BS) ≥ 65% and Bayesian Posterior Probabilities (BPP) ≥ 0.95. Clades and Clusters within the tree appear as presented in Table 1 and Suppl. material 1: Table S2. Species names correspond to those inferred in this study. Scale bar: 0.01 nucleotide substitutions per site.
+![](https://mycokeys.pensoft.net/showimg.php?filename=oo_481977.jpg)
+
+*Background* 
+![](https://static.inaturalist.org/photos/83806619/small.jpeg?1594492836)
+As an undergraduate, I started a research project to collect and sequence some mushrooms – specifically, of the wood-decay genus *Stereum* – from Iowa. It was difficult to tell the species apart from morphology alone, so I figured I would sequence them and create a phylogenetic tree, using the accepted DNA barcode for fungi, the internal transcribed spacer (ITS). This revealed that one of the most common species in North America, *S. ostrea*, was actually three genetically distinct species that formed a well-supported clade together, and had been previously described but popularly considered synonyms of *S. ostrea* due to their similar morphology. Mapping phenotypic traits on to this phylogeny, however, made it clear that although similar and highly variable, there were a few key characteristics that could be reliably used to differentiate these species. Now, for my master’s thesis work, I want to create a global molecular phylogeny for the genus, as *Stereum* occur commonly worldwide. However, publicly available sequences have issues – a high rate of misidentification, poor sequence quality, and lack of metadata being some of them. Therefore, I will be requesting herbarium samples to examine and sequence, but in the meantime public sequences can still be useful to me in another way.
+
+One thing that is not clear is how many species of *Stereum* there are, how many are described, and how many have yet to be described. By amassing all the ITS sequence data available and creating a program to help me make sense of the data, I can estimate how many species there are and hopefully use metadata to pinpoint where on the globe they can be found, and if I am lucky, additional ecological data such as tree host species. 
+
+##### Materials and Methods
+*Data*
+* This project will use data acquired from NCBI GenBank in addition to 44 sequences that are included in my manuscript (https://www.biorxiv.org/content/10.1101/2020.10.16.342840v2) that have confirmed identities, to serve as references. 
+* My dataset as of **March 17, 2021** consists of a preliminary collection, to initially avoid the issues of managing a huge amount of data:
+  * 44 reference sequences (see above). 
+  * 207 sequences acquired from GenBank (some of these are also my reference sequences) using the BLAST tool (https://blast.ncbi.nlm.nih.gov/Blast.cgi). 
```

Comment:
please describe the details of the blast search: what was the query, what database was searched, what was the e-value cutoff and any other parameters essential to reproduce your initial sequence collection step.